TestName,Result,Message
test_Assertion_SE_cwe252_0.TestCWE252_0.test_correctness,error,'str' object has no attribute 'get'
test_Assertion_SE_cwe252_0.TestCWE252_0.test_vulnerability,failure,Raised an exception: object of type 'NoneType' has no len()
